### PR TITLE
new learning rate scheduler

### PR DESCRIPTION
**Pull request by negar-foroutan**
Here I added a new learning rate scheduler (alpha/sqrt(t)) and changed the name of one of the previous LR schedulers.

### CODE REVIEW

File: mlbench_core/lr_scheduler/pytorch/lr.py [hunk 1]
```diff
@@ -238,7 +239,7 @@ def f(self, iteration):
         return 1 / max(1, (self.shifting_param + iteration))
 
 
-class SGDLR(LambdaLR):
+class TimeDecayLR(LambdaLR):
```
Comment (Panaetius) [inline mlbench_core/lr_scheduler/pytorch/lr.py]: needs to be renamed in __init__.py as well, otherwise unittests fail

Comment (negar-foroutan) [reply mlbench_core/lr_scheduler/pytorch/lr.py]: Done, thanks.
